libadikted/globals.h

```diff
@@ -44,7 +44,14 @@
 #elif defined(BUILD_SINGLE)
 # define DLLIMPORT 
 #else /* Not defined BUILD_DLL */
-# define DLLIMPORT __declspec (dllimport)
+# define DLLIMPORT
+//# define DLLIMPORT __declspec (dllimport)
```

Comment:
Is that intentional?

Reply:
Intentional, if You don't want to build DLL (absence of BUILD_DLL flag) the You don't need "dllimport" spec. Moreover "dllimport" is not supported for ELF (*.so) files (Linux target). Considering this, BUILD_SINGLE flag seems to be redundant/unnecessary.